src/summary/LongitudinalTable.scss

```diff
@@ -6,45 +6,129 @@
     text-align: center;
     min-height: 0px;
 }
-
 #longitudinal-table {
-    .left-table {
-        float: left;
-        .name {
-            max-width: 135px;
-            text-overflow: ellipsis;
-            overflow: hidden;
-            white-space: nowrap;
+    .vertical-scroll {
+        overflow-y: scroll !important;
+        max-height: 200px;
+        .sticky-table .sticky-table-row:nth-child(-n + 2) .sticky-table-cell {
+            //row
+            position: -webkit-sticky;
+            position: sticky;
+            top: 0 !important;
+            z-index: 2;
+        }
+        .sticky-table .sticky-table-row .sticky-table-cell:nth-child(-n + 4) {
+            //column
+            position: -webkit-sticky;
+            position: sticky;
+            left: 0;
+            z-index: 2;
+        }
+        .sticky-table
+            .sticky-table-row:nth-child(-n + 2)
+            .sticky-table-cell:nth-child(-n + 4) {
+            //row and column
+            position: -webkit-sticky;
+            position: sticky;
+            top: 0;
+            left: 0;
+            z-index: 3;
         }
-        .star-cell {
-            padding-left: 0px !important;
-            padding-right: 0px !important;
-            text-align: center !important;
-            min-width: 30px !important;
-            .star-clicked {
-                color: $interface-blue;
-            }
-            .star-hovered {
-                color: $interface-blue;
-            }
-            &.star-body {
-                &:hover {
-                    cursor: pointer;
+        .sticky-table {
+            border-collapse: collapse;
+            width: auto;
+            margin-top: 10px;
+            font-size: $size-m;
+            overflow-x: unset !important;
+
+            &.white-scrollbar::-webkit-scrollbar-thumb {
+                background-color: #ffffff !important;
+            }
+            #sticky-header {
+                top: 0 !important;
+            }
+            #sticky-name {
+                left: 50px;
+                min-width: 135px;
+                max-width: 135px;
+            }
+            #sticky-unit {
+                left: 185.56px;
+                min-width: 95px;
+            }
+            #section-header {
+                //starred or all labs/vitals
+                font-size: 0.75rem !important;
```

Comment:
#section-header and .header should be using variables, both $body-gray and $weight-normal. No font-family is necessary. The font-sizes are $size-xs and $size-m